src/DurableTask.SqlServer.AzureFunctions/SqlTargetScaler.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace DurableTask.SqlServer.AzureFunctions
+{
+    using System.Threading.Tasks;
+    using Microsoft.Azure.WebJobs.Host.Scale;
+
+#if NETCOREAPP
+    class SqlTargetScaler : ITargetScaler
+    {
+        readonly SqlMetricsProvider sqlMetricsProvider;
+        readonly TargetScalerResult scaleResult;
+
+        public SqlTargetScaler(string functionId, SqlMetricsProvider sqlMetricsProvider)
+        {
+            this.sqlMetricsProvider = sqlMetricsProvider;
+            this.scaleResult = new TargetScalerResult();
+            this.TargetScalerDescriptor = new TargetScalerDescriptor(functionId);
+        }
+
+        public TargetScalerDescriptor TargetScalerDescriptor { get; private set; }
```

Comment:
nit: remove the `private set` to make this a read-only property. You'll still be able to assign it from the constructor.